oap-server/server-alarm-plugin/src/main/java/org/apache/skywalking/oap/server/core/alarm/provider/RunningRule.java

```diff
@@ -290,28 +278,28 @@ private boolean isMatch() {
 
                 switch (valueType) {
                     case LONG:
-                        long lvalue = ((LongValueHolder)metrics).getValue();
+                        long lvalue = ((LongValueHolder) metrics).getValue();
                         long lexpected = RunningRule.this.threshold.getLongThreshold();
                         if (op.test(lexpected, lvalue)) {
                             matchCount++;
                         }
                         break;
                     case INT:
-                        int ivalue = ((IntValueHolder)metrics).getValue();
+                        int ivalue = ((IntValueHolder) metrics).getValue();
                         int iexpected = RunningRule.this.threshold.getIntThreshold();
                         if (op.test(iexpected, ivalue)) {
                             matchCount++;
                         }
                         break;
                     case DOUBLE:
-                        double dvalue = ((DoubleValueHolder)metrics).getValue();
+                        double dvalue = ((DoubleValueHolder) metrics).getValue();
                         double dexpected = RunningRule.this.threshold.getDoubleThreshold();
                         if (op.test(dexpected, dvalue)) {
                             matchCount++;
                         }
                         break;
                     case MULTI_INTS:
-                        int[] ivalueArray = ((MultiIntValuesHolder)metrics).getValues();
+                        int[] ivalueArray = ((MultiIntValuesHolder) metrics).getValues();
```

Comment:
What change the format of all these cases?

Reply:
alibaba coding guideline plugin did these coding format automatically..

Reply:
But we are not using it .SkyWalking has its own format, please revert those.